guides/v2.3/graphql/reference/quote.md

```diff
@@ -27,12 +160,413 @@ mutation {
 
 **Response**
 
-The response is the quote ID, which is sometimes called the cart ID.
+The response is the quote ID, which is sometimes called the cart ID. The remaining examples in this topic will use this cart ID.
 
 ```json
 {
   "data": {
-    "createEmptyCart": "6XZA7q1ooLEI0jLz8DfFrfruEqgxGzlt"
+    "createEmptyCart": "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C"}
   }
 }
 ```
+
+### Adding and removing coupons from a cart
+{:.no_toc}
+
+You can use mutations to add or remove coupons from a specified cart.
+
+### Coupon attributes
+The coupon object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cart_id` | String | The unique ID that identifies the customer's cart
+`coupon_code` | String | The coupon code
+
+### Apply coupon to cart
+
+Adds a coupon code to a cart.
+
+#### Syntax
+
+`mutation: applyCouponToCart`
+
+#### Example usage
+
+The following call adds a coupon code called `test2019` to a cart.
+
+**Request**
+
+``` text
+mutation {
+  applyCouponToCart(input: {cart_id: "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C", coupon_code: "test2019"}) {
+    cart {
+      applied_coupon {
+        code
+      }
+    }
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "applyCouponToCart": {
+      "cart": {
+        "applied_coupon": {
+          "code": "test2019"
+        }
+      }
+    }
+  }
+}
+
+```
+
+### Remove coupon from cart
+
+Removes a coupon from the specified cart.
+
+#### Syntax
+
+`mutation: removeCouponFromCart`
+
+#### Example usage
+
+The following example removes a coupon from the cart.
+
+**Request**
+
+``` text
+mutation {
+  removeCouponFromCart(input: {cart_id: "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C"}) {
+    cart {
+      applied_coupon {
+        code
+      }
+    }
+  }
+}
+```
+
+**Response**
+
+```json
+{
+  "data": {
+    "removeCouponFromCart": {
+      "cart": {
+        "applied_coupon": {
+          "code": "test2019"
+        }
+      }
+    }
+  }
+}
+```
+
+### Adding products to cart
+{:.no_toc}
+
+Adds simple items to a specific cart.
+
+### Add simple products to cart attributes
+The Add simple products to cart object can contain the following attributes:
+
+Attribute |  Data Type | Description
+--- | --- | ---
+`cartItems` | [SimpleProductCartItemInput] | The list of items to add to the cart
+`cart_id` | String | The unique ID that identifies the customer's cart
+
+#### Syntax
+
+`mutation: addSimpleProductsToCart`
+
+#### Example usage
+
+The following example adds two Joust Duffle Bags to the cart.
+
+**Request**
+
+``` text
+mutation {  
+  addSimpleProductsToCart(
+    input: {
+      cart_id: "4JQaNVJokOpFxrykGVvYrjhiNv9qt31C", 
+      cartItems: [
+        {
+          data: {
+            qty: 2
+            sku: "24-MB01"
+          }
+        }
+      ]
+    }
+  ) {
+    cart {
+      cart_id
```

Comment:
Let's extend the output so that we have confirmation that the items were added.

Reply:
added product.name and quantity